variant.go

```diff
@@ -4,10 +4,14 @@ type Variant struct {
 	BuildName        string                  `json:"name,omitempty" yaml:"name,omitempty"`
 	BuildDisplayName string                  `json:"display_name,omitempty" yaml:"display_name,omitempty"`
 	BatchTimeSecs    int                     `json:"batchtime,omitempty" yaml:"batchtime,omitempty"`
+	CronBatchTime    string                  `yaml:"cron,omitempty" bson:"cron,omitempty"`
+	Stepback         bool                    `yaml:"stepback,omitempty" bson:"stepback,omitempty"`
```

Comment:
I think this should be `*bool` because if using just a non-pointer bool, `omitempty` will omit the field entirely if it's set to false. The [task model uses *bool](https://github.com/evergreen-ci/shrub/blob/65b9d72a871ed1cd8e60aa69b0f0969da0030784/task.go#L10-L15) for this reason